src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-share-form/smb-share-form.component.html

```diff
@@ -0,0 +1,238 @@
+<div cdsCol
+     [columnNumbers]="{ md: 4 }">
+  <form name="smbShareForm"
+        #formDir="ngForm"
+        [formGroup]="smbShareForm"
+        novalidate>
+    <div i18n="form title"
+         class="form-header">
+      {{ action | titlecase }} {{ resource | upperFirst }}
+    </div>
+
+    <!-- Share Id -->
+    <div class="form-item">
+      <cds-text-label
+        for="share_id"
+        i18n
+        helperText="Unique share identifier"
+        i18n-helperText
+        cdRequiredField="Share Name"
+        [invalid]="smbShareForm.controls.share_id.invalid && smbShareForm.controls.share_id.dirty"
+        [invalidText]="shareError"
+        >Share Name
+        <input
+          cdsText
+          type="text"
+          id="share_id"
+          formControlName="share_id"
+          [invalid]="smbShareForm.controls.share_id.invalid && smbShareForm.controls.share_id.dirty"
+        />
+      </cds-text-label>
+      <ng-template #shareError>
+        <span
+          class="invalid-feedback"
+          *ngIf="smbShareForm.showError('share_id', formDir, 'required')"
+          i18n
+          >This field is required.</span
+        >
+      </ng-template>
+    </div>
+
+    <!-- Volume -->
+    <div class="form-item">
+      <cds-select
+        formControlName="volume"
+        for="volume"
```

Comment:
```suggestion
```
id is being used as `for` in this component. Its difficult to wrap our head but :carbon: 🤷🏿